lib/puppet/provider/netapp_security_login/cmode.rb

```diff
@@ -59,7 +59,7 @@ def flush
     when :present
       #if lock status changes, do the right thing
       if @original_values[:is_locked] != is_locked
-        if is_locked
+        if is_locked == "true"
```

Comment:
@hunner on looking at the code again, we recieve this as a string from netapp. and when i explicitly set the type to be boolean, Original_values[:is_locked] does not return a boolean either. 

it has to be the way it is. otherwise we have to start explicitly casting all over the place